Reference/conformance/conformance.pb.swift

```diff
@@ -98,28 +98,20 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message {
 
   var protobufPayload: Data {
     get {
-      if case .protobufPayload(let v)? = payload {
-        return v
-      }
+      if case .protobufPayload(let v)? = payload { return v }
       return Data()
```

Comment:
Someday, we should look at `Data` and message-valued fields here and have them return a static empty object instead of constructing a new one.